benchmark/url/url-searchparams-common.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+exports.inputs = {
```

Comment:
The convention seems to be to include this type of data in all benchmark files, so I did that instead.